gammapy/estimators/asmooth_map.py

```diff
@@ -135,14 +136,15 @@ def _sqrt_ts_cube(cubes, method):
             )
         return scube
 
-    def run(self, dataset):
+    def run(self, dataset, show_pbar=False):
```

Comment:
I think `show_pbar` could be rather considered as configuration and I would recommend to move it to `Estimator.__init__` consistently.

Reply:
To avoid replication of the argument in all `__init__` shouldn't we move it to the base class instead?